spec/jobs/google/search_keyword_job_spec.rb

```diff
@@ -0,0 +1,130 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Google::SearchKeywordJob, type: :job do
+  include ActiveJob::TestHelper
+
+  describe '#perform' do
+    context 'given a valid request' do
+      it 'saves 3 as top_ads_count in the Database' do
+        stub_request(:get, %r{google.com/search}).to_return(body: file_fixture('html/valid_google.html').read)
+        keyword = Fabricate(:keyword)
+
+        described_class.perform_now keyword
```

Comment:
For testing the job, you can reference this guide.  https://guides.rubyonrails.org/testing.html#testing-jobs

Reply:
Thank you, I added to the controller spec. Will make sure to use it in the Background job task 👍